server/store/memory/string_store.go

```diff
@@ -59,3 +60,7 @@ func (ss *stringStore) RemoveString(s string) error {
 
 	return nil
 }
+
+func (ss *stringStore) Stop() <-chan error {
+	return stopper.AlreadyStopped
```

Comment:
Are we sure we don't want this to throwaway the current state of the internal map of strings? Thus the next GC would free all of that memory. By not doing so, we're making an assumption about the lifetime of the program.